pkg/services/serviceaccounts/extsvcaccounts/metrics.go

```diff
@@ -24,14 +24,14 @@ func newMetrics(reg prometheus.Registerer, saSvc serviceaccounts.Service, logger
 		prometheus.GaugeOpts{
 			Namespace: metricsNamespace,
 			Name:      "extsvc_total",
```

Comment:
Nit: should we also rename the metric or do you think it's too late for that?

Reply:
I'd rather keep the name as it is. As of today external service accounts are the only managed service accounts, but that might not be true tomorrow. Or am I overthinking this?

Reply:
Ah, interesting, I didn't realise that we might have other types of external SAs in the future. Do we have any concrete plans?

Reply:
> Do we have any concrete plans

No :smile: hence the:

> Or am I overthinking this?

I was thinking of this like roles, at the beginning we only had `fixed` roles, but then came `managed`, `extsvc`, `plugins`... All of these are technically managed by Grafana, and this is all the frontend should care about. Yet they have different naming patterns in the backend.

Reply:
After some conversations around the topic we will not rename anything `managed` but instead `external`. We feel we're thinking way ahead of time.

Reply:
I'll be undoing some of the renames